### ISSUE TITLE
Support for Razor (*.cshtml)

### ISSUE THREAD

**Issue by naando-araujo** [pranaygp/vscode-css-peek#75]
Hi, I'm using the default config, and just tried to add 'cshtml' or 'razor'.
After I saved, I restarted my vscode but it was not able to find any symbol by class name.

```
"css_peek.peekFromLanguages": [
  "html",
  "ejs",
  "erb",
  "php",
  "javascriptreact",
  "typescriptreact",
  "typescript",
  "javascript",
  "razor"
]
```

Thanks for your time.

**Comment by pranaygp** [comment]
Hmm, it seems like adding `razor` should work. Can you give me an example I can use to reproduce this issue?

Even better would be a tiny PR to this repo with a test case for `.cshtml` that fails. It's really easy to add such a test
1. Add a .cshtml file in [this folder](https://github.com/pranaygp/vscode-css-peek/tree/master/tests/fixture)
2. Add a test case in this file, just like this one, that fails for the .cshtml file. https://github.com/pranaygp/vscode-css-peek/blob/f5968daa4ed1faa6428d94ade84a496aa04ad581/tests/src/suite/extension.test.ts#L57-L65

**Comment by anuj-chauhan-2020** [comment]
I am not able to use peek definition in ASP.NET CORE 3.1 MVC project in which views are there . I am trying to navigate the css class name from a view file .cshtml but it says no definitions found for the <classname> . This extension is working pretty good in normal html files though.

**Comment by pranaygp** [comment]
This works just fine for me:

![image](https://user-images.githubusercontent.com/1797812/93686840-f494e780-fa6d-11ea-9f5e-c2b47be6f291.png)

Make sure you're on the latest version of CSS Peek (4.0.0).

**Comment by anuj-chauhan-2020** [comment]
Attaching the folder hierarchy we use in .Net Core Web App in which it is not working . I have used your html and css file contents from tests folder . Please check 
![Untitled](https://user-images.githubusercontent.com/71555479/94399701-80e98d80-0185-11eb-89a7-e598be98cdb1.png)

**Comment by TerminalZero** [comment]
Yes, as of version 4.2.0, "no definition could be found" persists .cshtml files in 2023. It's just not a supported file type, regardless of the settings added.
